roles/nvidia-driver/tasks/main.yml

```diff
@@ -27,6 +27,7 @@
   apt:
     name: "{{ item }}"
     install_recommends: no
+    dpkg_options: "force-overwrite"
```

Comment:
Yuck. What does Ubuntu OS come preinstalled with?

Reply:
Can we just ensure that a specific package is absent before running this command?

Reply:
Kubespray installs `18.06.1~ce~3-0~ubuntu` in the `container-engine/docker : ensure docker packages are installed` task. When installing nvidia-docker, it complains that it needs `5:18.09.2~3-0~ubuntu-bionic`.

Reply:
Which package are you referring to? docker-ce? Wouldn't the change need to go into either the docker or nvidia-docker role instead of this one, then? Sorry, I think I'm missing something.

Reply:
I'm going back and replaying it. I vaguely remember so *.so which could not get overwritten. There is also the conflict between the versions of docker-ce, which I alluded to above. Perhaps the latter is what is causing the former.

Reply:
Removed this since it _should_ work with the nvidia-docker changes.